rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+This flexibility is critical in a decentralized ecosystem where the diversity of users and use cases demands a more adaptable privacy solution.
+
+In the NSSA model, users have the autonomy to select how they interact with decentralized applications (Dapps) by choosing from four types of transaction executions: **public, 
+private, shielded, and deshielded**. This model allows users to tailor their privacy settings on a per-transaction basis, selecting the most appropriate execution type for each 
+specific interaction. For instance, a user concerned about data confidentiality might opt for a fully private transaction, while another user, wary of privacy but seeking transparency, 
+might choose a public execution.
+
+While selective privacy may appear complex, especially for users who are not technically inclined, Nescience mitigates this by allowing the community or developers to 
+establish best practices and recommended approaches. These guidelines provide users with an informed starting point, and over time, users can adjust their privacy 
+settings as their preferences and trust in the platform evolve. Importantly, selective privacy gives users the right to alter their privacy level at any point in the future, 
+ensuring that their privacy settings remain aligned with their needs as they change.
+
+This approach not only empowers users but also facilitates greater adoption of Dapps. Users who are skeptical about privacy concerns can initially engage with transparent 
+transactions and gradually shift towards more private executions as they gain confidence in the system, and vice versa for users who start with privacy but later find transparency 
+beneficial for certain transactions. In this way, selective privacy bridges the gap between privacy and transparency, allowing for an optimal balance to emerge from the community’s 
+collective preferences.
+
+To liken this to open-source projects: in traditional systems, developers fix privacy rules much like immutable code—users must comply with these fixed settings. 
+In contrast, with selective privacy, the rules are malleable and shaped by the users’ preferences, enabling the community to find the ideal balance between privacy and efficiency over time.
+
+NSSA is distinct from traditional zero-knowledge (ZK) roll-ups in several key ways. One of the unique features of NSS is its **public execution type**, which does not 
+require Zero-Knowledge Proofs or a zkVM. This provides a significant advantage in terms of scalability and efficiency, as users can choose public executions for 
+transactions that do not require enhanced privacy, avoiding the overhead associated with ZKP generation and verification.
+
+Moreover, NSSA introduces two additional execution types—**shielded and deshielded**—which further distinguish it from traditional privacy-preserving roll-ups. 
```

Comment:
added double * after "shielded" and before "deshielded" to make only those terms bold and not the "and"